src/requirements/reqs-functions.ts

```diff
@@ -284,7 +284,10 @@ export function computeRequirements(
       // userChoiceOnFulfillmentStrategy
       correspondingRequirement: requirementsToBeConsideredInGraph[0],
       coursesOfChosenFulfillmentStrategy: []
-    })
+    }),
+    // TODO: Replace this dummy implementation once we decided how to determine if a requirement is
+    // double-countable. Meanwhile, make it always return true to match the old behavior.
+    allowDoubleCounting: () => true
```

Comment:
If we decided false was the default, would this function just return true if we specifically say a requirement can be double counted in our checkers or if it was a minor requirement?
And do you want to discuss how we want to handle this over Slack with maybe Ein and/or Han?

Reply:
Yes